mysql-test/suite/rocksdb/t/blind_delete_without_tx_api.test

```diff
@@ -0,0 +1,131 @@
+--source include/have_rocksdb.inc
+
+source include/master-slave.inc;
+
+--source include/rpl_connection_master.inc
+
+set @save_rocksdb_blind_delete_primary_key=@@session.rocksdb_blind_delete_primary_key;
```

Comment:
There is no need to save/restore SESSION variables. This technique is only needed for GLOBAL ones.